pkgs/development/go-modules/generic/default.nix

```diff
@@ -55,11 +58,24 @@
 with builtins;
```

Comment:
```suggestion
with lib;
```

Reply:
Let's keep this PR focused on the vendor changes.